store/tikv/2pc.go

```diff
@@ -1374,15 +1375,50 @@ func (c *twoPhaseCommitter) stripNoNeedCommitKeys() {
 	c.mutations = m.subRange(0, newIdx)
 }
 
+const amendableType = uint64((1 << model.ActionAddColumn) | (1 << model.ActionDropColumn))
```

Comment:
What happen if there are two DDL changes that drop the column and than add back the column?

Reply:
It's ok the newly added column will use the default value